ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,21 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-provision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-disconnected-private-provision
+  - ref: workers-rhel-aws-provision
+  - ref: workers-rhel-pre-hook-sshtunnel
+  - chain: workers-rhel
```

Comment:
Per yesterday's meeting, suggest to append `ref: mirror-images-enable-qe-catalogsource` here.

Reply:
already added in `cucushift-installer-rehearse-aws-ipi-disconnected-private-provision`